pkg/operators/formatters/formatters.go

```diff
@@ -268,40 +272,51 @@ var replacers = []replacer{
 		replace: func(logger logger.Logger, ds datasource.DataSource, in datasource.FieldAccessor) (func(data datasource.Data) error, error) {
 			// We do some length checks in here - since we expect the in field to be part of an eBPF struct that
 			// is always sized statically, we can avoid checking the individual entries later on.
-			in.SetHidden(true, false)
-			ips := in.GetSubFieldsWithTag("type:gadget_ip_addr_t")
+			ips := in.GetSubFieldsWithTag("type:" + IPAddrTypeName)
 			if len(ips) != 1 {
-				return nil, fmt.Errorf("expected %d gadget_ip_addr_t field, got %d", 1, len(ips))
+				return nil, fmt.Errorf("expected %d %q field, got %d", 1, IPAddrTypeName, len(ips))
 			}
 			if ips[0].Size() != 16 {
-				return nil, fmt.Errorf("expected gadget_ip_addr_t to have 16 bytes")
+				return nil, fmt.Errorf("expected %q to have 16 bytes, got %d", IPAddrTypeName, ips[0].Size())
 			}
 			versions := in.GetSubFieldsWithTag("name:version")
 			if len(versions) != 1 {
 				return nil, fmt.Errorf("expected exactly 1 version field")
 			}
-			out, err := in.AddSubField("string", api.Kind_String, datasource.WithTags("l3string"))
+			// Pretty L3 address
+			addrName := strings.TrimSuffix(ips[0].Name(), "_raw")
+			addrF, err := in.AddSubField(addrName, api.Kind_String,
+				datasource.WithFlags(datasource.FieldFlagHidden),
```

Comment:
Is `datasource.WithFlags(datasource.FieldFlagHidden)` necessary when all subfields are hidden couple lines later?

Reply:
True, removed it in the latest update.